src/olympia/addons/models.py

```diff
@@ -285,6 +285,8 @@ class Addon(OnChangeMixin, ModelBase):
                                         db_column='bayesianrating')
     total_reviews = models.PositiveIntegerField(default=0,
                                                 db_column='totalreviews')
+    text_reviews = models.PositiveIntegerField(default=0,
```

Comment:
`text_reviews_count` ? `text_reviews` sound more like a list of text reviews that a count (I wish `total_reviews` was called `reviews_count` but I'm too scared to change it...)